tsconfig.json

```diff
@@ -0,0 +1,9 @@
+{
```

Comment:
Don't add this file. Even if it's needed, we need to think thru the details of adding tsconfig.json (which we haven't)

Reply:
OK